src/modals/ClaimPayouts/Overview.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { ButtonSubmit, ModalNotes } from '@polkadot-cloud/react';
+import { forwardRef } from 'react';
+import { usePayouts } from 'contexts/Payouts';
+import BigNumber from 'bignumber.js';
+import { useTranslation } from 'react-i18next';
+import { Item } from './Item';
+import { ContentWrapper } from './Wrappers';
+import type { OverviewProps } from './types';
+
+export const Overview = forwardRef(
+  ({ setSection, setPayouts }: OverviewProps, ref: any) => {
+    const { t } = useTranslation('modals');
+    const { unclaimedPayouts } = usePayouts();
+
+    const claimAllPayouts = Object.entries(unclaimedPayouts || {}).map(
+      ([era, validatorPayout]) => ({
+        era,
+        payout: Object.entries(validatorPayout)
+          .reduce(
+            (acc: BigNumber, [, amount]) => acc.plus(amount),
+            new BigNumber(0)
+          )
+          .toString(),
+        validators: Object.keys(validatorPayout),
+      })
+    );
+
+    return (
+      <ContentWrapper>
+        <div className="padding" ref={ref}>
+          <div style={{ margin: '1rem 0 0.5rem 0' }}>
+            <ButtonSubmit
+              disabled={Object.values(unclaimedPayouts || {}).length === 0}
+              text={t('claimAll')}
```

Comment:
I am not able to claim all payouts for `//Alice` user when connected to Devnet. I already cliamed single era 8 payout (this worked), however 1) view still shows era 8 to be paid but as 0 reward and 2) the `Claim All ` results in `Transaction was cancelled.



![image](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/assets/3909333/ff7a72b5-2a16-45c1-b6ba-a714475fc69f)

![image](https://github.com/Cardinal-Cryptography/aleph-zero-dashboard/assets/3909333/fd131143-636c-4864-97f7-80f4c0ede0ce)

Reply:
Turns out UI tries to batch a few `staking.payoutStakers` call and that won't fit in our block. Let's hide `Claim All` option for now.

Reply:
done